### DISCUSSION TITLE
Able to change to date format

### DISCUSSION THREAD

**Discussion by D3an1el** [ellite/Wallos#660] [Ideas]
Please have an option to change date format from mm/dd/yy to dd/mm/yy for  countries that use date/month/year format

**Comment by ellite** [comment]
Hi. It uses the locale settings from your browser.
It already works with that format.

![image](https://github.com/user-attachments/assets/c2c0e961-c41c-4f1f-aba3-ceb64fa07e98)

**Reply by Salvora** [reply]
Any way to forcefully change this regardless of the browser locale?

**Comment by D3an1el** [comment]
Thanks for your prompt response. To prevent bowser fingerprinting brave browser and librewolf stops this.

![locale finger printing](https://github.com/user-attachments/assets/a44e3d42-8a8f-4391-93a7-fa46199e3c1b)
